test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AsyncCancellationEndToEndTests.cs

```diff
@@ -37,6 +37,7 @@ public AsyncCancellationEndToEndTests()
 
             _host = new HostBuilder()
                 .ConfigureDefaultTestHost<AsyncCancellationEndToEndTests>()
+                .AddAzureStorage()
```

Comment:
ConfigureDefaultTestHost already has a call to AddAzureStorage inside of it. Why is this needed everywhere?